### PR TITLE
Add new mud/dirt turfs, rough wood planks, new chest sprites, etc

### PR DESCRIPTION
**Pull request by noelle-lavenza**
## Description of changes
- adds new mud/dirt turfs, sprites originally by InfraredBaron, I have been given permission to use them by the commissioner (and they are under CC-BY-SA)
- adds rough wood plank tiles, easier to craft but more wasteful, with a rougher-looking sprite, from the same source as the above
- adds Doe's new chest sprites
- adds doc comments to lighting_atom
- fixes being unable to fill containers from casks
- adjusts the color ramp for normal wood (not chipboard laminate) floor tiles
- replaces some wood floor tiles on shaded hills maps with rough counterparts
- gives the shrine mahogany floors in the sanctuary to make it more visibly Special, and to increase the contrast with the rough walnut floors in the other rooms
- fixes cooking containers cooking things other than soup without needing heat

## Why and what will this PR improve
visual changes and improvements, plus more turf variety. also some bugfixes

## Authorship
sprites by infraredbaron, given permission by the commissioner (AlcoholismIncarnate). code by me

## Changelog
:cl: ophelia v0.8
add: added rough wood floor tiles, easier to craft but more wasteful
imageadd: added new dirt and mud tile sprites
imageadd: added new wooden chest sprites, by Doe
tweak: mud and soil plots are now properly greyscaled to soil material color
/:cl:

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Would like to see screenshots of the new turfs. Are they in use elsewhere?

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 1]
```diff
@@ -323,7 +323,22 @@
 	closet_appearance = /decl/closet_appearance/crate/chest
 	material_alteration = MAT_FLAG_ALTERATION_COLOR | MAT_FLAG_ALTERATION_NAME | MAT_FLAG_ALTERATION_DESC
 	material = /decl/material/solid/organic/wood/oak
+	reinf_material = /decl/material/solid/metal/iron
```
Comment (MistakeNot4892) [inline code/game/objects/structures/crates_lockers/crates.dm]: This is going to mean chests can never be made craftable.

Comment (noelle-lavenza) [reply code/game/objects/structures/crates_lockers/crates.dm]: they already are, that's why it's failing ci

Comment (noelle-lavenza) [reply code/game/objects/structures/crates_lockers/crates.dm]: could do, in the future, a crafted subtype with no trim and you have to use a rod or something to apply trim. that or the crafting recipe could be expanded, e.g. wooden box (storage item) plus lid plus metal trim/hinges = wooden chest with metal trim. iunno. either way it passes CI now

**Comment by out-of-phaze** [comment]
as discussed on discord, the dirt sprites need to be reverted, they're too low contrast and look a bit too kitbashed

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**